activestorage/test/template/image_tag_test.rb

```diff
@@ -0,0 +1,40 @@
+require "test_helper"
+require "database/setup"
+
+class User < ActiveRecord::Base
+  has_one_attached :avatar
+end
+
+class ActiveStorage::ImageTagTest < ActionView::TestCase
```

Comment:
Should they go under `test/activerecord` and test with generic models or under `test/activestorage` and stay as is, with Blobs and stuff?